src/QMCDrivers/QMCCostFunctionOMP.cpp

```diff
@@ -709,6 +709,36 @@ void QMCCostFunctionOMP::resetPsi(bool final_reset)
     psiClones[i]->resetParameters(OptVariablesForPsi);
 }
 
+///////////////////////////////////////////////////////////////////////////////////////////////////
+/// \brief  For each component of the trial function, copy that component's optimizable parameters
+///         into it, convert the parameters to a standard form (for objects that need this, most
+///         don't) and then optionally copy the standard form parameters back.
+///
+///////////////////////////////////////////////////////////////////////////////////////////////////
+void QMCCostFunctionOMP::putParametersInStandardForm() {
+
```

Comment:
The commit log shows this is a hack and can be fixed with the usual way of QMCPACK checking in/out paramaters.